### ISSUE TITLE
Expliciently mention that calculating ∇C is only introduced later on

### ISSUE THREAD

**Issue by alexander255** [mnielsen/neural-networks-and-deep-learning#37]
Hi, thanks for the great book! I've just managed to complete chapter 1!

One thing that really confused me through is that you, in the *Learning with gradient descent* section, talk a lot about how we're going to use ∇C for everything, but never actually mention how we obtain its value. Having read on, I now know, of course, that ∇C is obtained using the backpropagation algorithm (that is formally introduced in the chapter 2), but I did waste a lot of time reading the gradient descent section over and over, trying to figure out what I missed.

Probably the right place to mention that ∇C is something that we only use, not calculate yet, in the *Learning with gradient descent* section, would be somewhere close to this sentence:

> Summing up, the way the gradient descent algorithm works is to repeatedly compute the gradient ∇C, and then to move in the opposite direction, "falling down" the slope of the valley. We can visualize it like this:

**Comment by mnielsen** [comment]
Glad you like the book.  I'm no longer making minor edits like this, however - I need to move on to other projects.

**Comment by alexander255** [comment]
Not even if I provide a pull request for it?

**Comment by mnielsen** [comment]
The book's source isn't on GitHub (I use a kludged-together LaTeX preprocessor).

**Comment by alexander255** [comment]
Hmm, makes sense that one don't want to fire up your home-grown "tool-chain" for every micro-edit, I guess. (Just read up on the status of the book's code…)
Out of curiosity: Why was the raw book TeX never published? (Along with a big "This will explode in your face, don't use unless you like pain"-banner, or something…)

Anyway: Big thanks for writing this book! :heart:
